ansible-allinone-demo-vm/roles/nginx/templates/nginx.conf

```diff
@@ -156,10 +156,16 @@ http {
         add_header Referrer-Policy strict-origin-when-cross-origin;
 
         # Basic open CORS for everyone
-        add_header Access-Control-Allow-Origin $http_origin;
-        add_header Access-Control-Allow-Methods 'GET, POST';
-        add_header Access-Control-Allow-Credentials true;
-        add_header Access-Control-Allow-Headers 'Origin,Content-Type,Accept,Authorization';
+        add_header Access-Control-Allow-Origin $http_origin always;
+        add_header Access-Control-Allow-Methods 'GET, POST' always;
```

Comment:
Isn't 'GET, POST, OPTIONS' necessary? That's what I thought up to now.

Reply:
I guess I can certainly add `OPTIONS`!